python/rikai/internal/reflection.py

```diff
@@ -22,3 +22,41 @@ def find_class(class_name: str):
     module, cls = class_name.rsplit(".", 1)
     mod = importlib.import_module(module)
     return getattr(mod, cls)
+
+
+def check_class(class_name: str):
+    """
+    Assuming `x.y.z.name` as class_name, check
```

Comment:
is what you're calling a class here actually a class? Aren't we using it to find functions?

Reply:
no

It just follows the convention of `find_class`